PocketKit/Sources/PocketKit/Home/RecommendationCell.swift

```diff
@@ -132,111 +113,44 @@ class RecommendationCell: UICollectionViewCell {
-            textStack.spacing = Mini.textStackSpacing
-            excerptLabel.isHidden = true
-            saveButton.isTitleHidden = true
-            subtitleLabel.numberOfLines = Mini.numberOfSubtitleLines
-
-            buttonStackTopConstraintHero.isActive = false
-            buttonStackTopConstraintMini.isActive = true
-        }
+    
+    override func layoutSubviews() {
+        contentView.layer.masksToBounds = false
+        layer.cornerRadius = 16
+        contentView.layer.cornerRadius = 16
+        contentView.layer.shadowColor = UIColor(.ui.border).cgColor
+        contentView.layer.shadowOffset = .zero
+        contentView.layer.shadowOpacity = 1.0
+        contentView.layer.shadowRadius = 6
+        contentView.layer.shadowPath = UIBezierPath(roundedRect: contentView.layer.bounds, cornerRadius: contentView.layer.cornerRadius).cgPath
+        contentView.layer.backgroundColor = UIColor(.ui.white1).cgColor
     }
 }
 
 extension RecommendationCell {
-    struct Hero {
-        static let textStackTopMargin: CGFloat = 16
-        static let buttonStackTopMargin: CGFloat = 10
-        static let textStackSpacing: CGFloat = 8
-        static let numberOfSubtitleLines = 1
-    }
-
-    struct Mini {
-        static let textStackTopMargin: CGFloat = 10
-        static let buttonStackTopMargin: CGFloat = 4
-        static let textStackSpacing: CGFloat = 4
-        static let numberOfSubtitleLines = 2
-    }
-
-    static let layoutMargins = UIEdgeInsets(top: 8, left: 8, bottom: 8, right: 8)
+    static let textStackTopMargin: CGFloat = 16
+    static let layoutMargins = UIEdgeInsets(top: 0, left: 16, bottom: 16, right: 16)
     static let imageAspectRatio: CGFloat = 9/16
     static let numberOfTitleLines = 3
-    static let numberOfExcerptLines = 3
-    static let saveButtonHeight: CGFloat = 21
+    static let numberOfSubtitleLines = 2
 }
 
 extension RecommendationCell {
-    static func miniHeight(viewModel: RecommendationCellViewModel, availableWidth: CGFloat) -> CGFloat {
-        let adjustedWidth = (availableWidth / 2).rounded(.down) - Self.layoutMargins.left - Self.layoutMargins.right
-        let imageHeight = adjustedWidth * Self.imageAspectRatio
-
-        let titleHeight = adjustedHeight(
-            of: viewModel.attributedTitle,
-            availableWidth: adjustedWidth,
-            numberOfLines: numberOfTitleLines
-        )
-
-        let detailHeight = adjustedHeight(
-            of: viewModel.attributedDetail,
-            availableWidth: adjustedWidth,
-            numberOfLines: Hero.numberOfSubtitleLines
-        )
-
-        return Self.layoutMargins.top
-        + imageHeight
-        + Mini.textStackTopMargin
-        + titleHeight
-        + Mini.textStackSpacing
-        + detailHeight
-        + Mini.buttonStackTopMargin
-        + Self.saveButtonHeight
-        + Self.layoutMargins.bottom
-    }
-
     static func fullHeight(viewModel: RecommendationCellViewModel, availableWidth: CGFloat) -> CGFloat {
```

Comment:
leaving height calculation code just in case useful context for when tackling dynamic fonts